source/includes/_samples.md

```diff
@@ -0,0 +1,66 @@
+# Sample Code
+
+## Resizing an `iframe` Based on Content Changes
+
+> iframe Code:
+
+```javascript
+document.body.style.overflow = 'hidden';
+
+function sendResizeMessage() {
+  window.parent.postMessage({
+    type: 'IFRAME_HEIGHT_RESIZE',
+    frameHeight: document.body.offsetHeight + 10 /* a little extra for good measure */
+  }, '*');
+}
+
+if (window.ResizeObserver) {
+  var heightObserver = new ResizeObserver(function() {
+    sendResizeMessage();
+  });
+  heightObserver.observe(document.body);
+} else {
+  // IE 11
+  setInterval(
+    sendResizeMessage,
+    500 /* ms */
+  );
+}
+```
+
+> Integration Code:
+
+```javascript
+(function (WIAPI) {
+  API = new WIAPI();
+
+  API.insert('content', function (elem, meta) {
+    var iframeElem = document.createElement('iframe');
+    iframeElem.src = '/path-to-iframe.htm';
+    iframeElem.classList.add('my-integration-name-iframe');
+    API.append(elem, iframeElem);
+  });
+
+  function setIframeHeight(e) {
+    if (e.origin !== 'https://yourdomain.com') {
+      // You should ALWAYS verify the origin matches the third party domain
+      // the iframe is loaded from. For more information, see:
+      // https://developer.mozilla.org/en-US/docs/Web/API/Window/postMessage#Security_concerns
+      return;
+    }
+
+    if (e.data.type === 'IFRAME_HEIGHT_RESIZE' && e.data.frameHeight) {
+      var iframes = document.getElementsByClassName('my-integration-name-iframe');
```

Comment:
If we had them pass the iframe class name into the message object, they could reuse this function for multiple iframes.

Reply:
Sure, I can make this update. I'll await a response on the other comment before proceeding since it could change the implementation a little.